hdf5_demo/hdf2gid.py

```diff
@@ -18,7 +18,8 @@
 nx  = sp[0]
 ny  = sp[1]
 nz  = sp[2]
-ix  = nx // 64
+sampling_rate = 8
+ix  = nx // sampling_rate
```

Comment:
Having `sampling_rate = 8` changes the previous default of `ix`. Is this intentional?

Reply:
I want to make it explicit, that whether 8 or 64, this is somewhat of a sampling rate. Basically it defines how large the GiD file will be. The generated H5 - used for simulating - will not change.